feature/bgp/multipath/otg_tests/bgp_multipath_wecmp_lbw_community_test/README.md

```diff
@@ -19,43 +19,117 @@ Validate BGP in multipath UCMP support with link bandwidth community
+        *   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/config/send-community-type
+        *   /network-instances/network-instance/protocols/protocol/bgp/global/use-multiple-paths/ebgp/link-bandwidth-ext-community/config/enabled
+    *   Advertise equal cost paths from port2 and port3 of ATE
+    *   Check entries in FIB for advertised prefix, it should have 2 entries.\
+        The `weight` leafs of next-hops shall be in 1:1 ratio.
+        *   /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/state/index
+        *   /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/state/weight
+    *   Initiate traffic from ATE port-1 to the DUT and destined to internal
+        target network
+    *   Check entire traffic should only be unequally forwarded between DUT
+        port2 and port3 in ratio 1:1
+
+*   [TODO] RT-1.52.3: Verify BGP multipath when some path missing link-bandwidth extended-community
+
+    *   Configure ATE port 1, 2 and 3 on different AS, with boths AFI/SAFI
+        * Advertise IPv4 and IPv6 internal target, both, form both ATE port-1 and ATE port-2 in eBGP.
+        * For ATE port 2 attach `link-bandwidth:23456:10K` extended-community
+        * For ATE port 3 **DO NOT** attach any link-bandwidth extended-community
+    *   Enable multipath, set maximum-paths limit to 2, enable allow multiple
+        AS, and send community type to [STANDARD, EXTENDED, LARGE]
+        *   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/use-multiple-paths/config/enabled
+        *   /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/use-multiple-paths/ebgp/config/allow-multiple-as
+        *   /network-instances/network-instance/protocols/protocol/bgp/globalp/afi-safis/afi-safi/use-multiple-paths/ebgp/config/maximum-paths
+        *   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/config/send-community-type
+        *   /network-instances/network-instance/protocols/protocol/bgp/global/use-multiple-paths/ebgp/link-bandwidth-ext-community/config/enabled
+    *   Advertise equal cost paths from port2 and port3 of ATE
+    *   Check entries in FIB for advertised prefix, it should have 2 entries.\
+        The `weight` leafs of next-hops shall be in 1:1 ratio.
+        *   /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/state/index
+        *   /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/state/weight
+    *   Initiate traffic from ATE port-1 to the DUT and destined to internal
+        target network
+    *   Check entire traffic should only be unequally forwarded between DUT
+        port2 and port3 in ratio 1:1
+
+*   [TODO] RT-1.52.4: Verify BGP multipath when none of paths has link-bandwidth extended-community
```

Comment:
Is this a duplicate of RT-1.51?

https://github.com/openconfig/featureprofiles/tree/9bf506f64578e39a185c4f0cb7386c453e1c3942/feature/bgp/multipath/otg_tests/bgp_multipath_ecmp_test

Reply:
indeed. removed.